introduction.md

```diff
@@ -1,2 +1,5 @@
 I'm Tayla, I'm a trainee software developer.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?